src/main/java/mods/railcraft/common/blocks/aesthetics/brick/BrickTheme.java

```diff
@@ -72,6 +73,12 @@ public void initRecipes(BlockBrick block) {
                     "III",
                     'I', new ItemStack(Blocks.ICE),
                     'L', "gemLapis");
+            CraftingPlugin.addRecipe(new ItemStack(block, 8, 2),
+                    "III",
+                    "ILI",
+                    "III",
+                    'I', new ItemStack(Blocks.PACKED_ICE),
+                    'L', "gemLapis");
```

Comment:
That's a possibility. But I want @CovertJaguar to put his opinion on that before I do it. He might have intentionally done that in the first place.